BackofficeBundle/Controller/ContentTypeController.php

```diff
@@ -77,22 +77,9 @@ public function newAction(Request $request)
         );
 
         $form->handleRequest($request);
-
-        if ($form->isValid()) {
-            $documentManager = $this->get('doctrine.odm.mongodb.document_manager');
-            $documentManager->persist($contentType);
-            $documentManager->flush();
-
-            $this->get('session')->getFlashBag()->add(
-                'success',
-                $this->get('translator')->trans('open_orchestra_backoffice.form.content_type.creation')
-            );
-
+        if (!$request->get('no_save')) {
+            $this->handleForm($form, $this->get('translator')->trans('open_orchestra_backoffice.form.content_type.creation'), $contentType);
             $this->dispatchEvent(ContentTypeEvents::CONTENT_TYPE_CREATE, new ContentTypeEvent($contentType));
-
-            return $this->redirect(
```

Comment:
If you save the content-type, it is more userfriendly that you can directly edit it without going on an other page.

It is the workflow set on all the others cruds